src/components/ListItem.jsx

```diff
@@ -46,6 +46,11 @@ export function ListItem({ item, listToken }) {
 		: sub(new Date(), { days: 1 });
 
 	const isRecentlyPurchased = sub(new Date(), { days: 1 }) < milliseconds;
+	const handleDelete = async () => {
+		if (window.confirm('Are you sure you want to delete this item?')) {
+			await deleteItem(listToken, item.id);
+		}
+	};
 
```

Comment:
Well done guys  this works as expected but can I suggest we build something more custom like a modal dialog box that appears at the center of the screen and blurs out every other thing on the page. This is what is used on most websites. A user clicks on something and a small dialog box appears in the middle of the page with prompts to either ok or cancel. That would look better than the javascript alert that only appears at the top of a page. To build the modal is not so difficult. If you choose to do this, you may need to create a new component to handle the modal dialog box and render it within the ListItem component or you can choose to build this and render it in the List component and use a state to show it or hide it. For the dialog box itself, you should create two containers, the first container will define how the modal appears. It should have a fixed position and the background should be blurred out or transparent. Then the second container will contain the content of the box. See an example of how this can look below:

![image](https://github.com/the-collab-lab/tcl-59-smart-shopping-list/assets/79487718/723e9c03-7ee5-411d-bdc5-6a96067f544f)

There are a ton of different ways to build a modal dialog box. You can find good suggestions on google.
Just a suggestion😉

Reply:
Thanks, Mike.
 We thought of this but we saw this as part of the notes : 
"It might be tempting to create your own modal dialog component to handle confirmation, but modals are fraught with accessibility concerns, so consider using a simple [JavaScript confirm dialog](https://developer.mozilla.org/en-US/docs/Web/API/Window/confirm)."

Do you still think we should create a modal?

Reply:
Do you still think we should create a modal?

Reply:
Ah! I guess this works then